core/kazoo_config/src/kapps_config_util.erl

```diff
@@ -49,9 +49,9 @@ load_config_from_reseller(Account, Config) ->
     ResellerId = kz_services:find_reseller_id(Account),
     load_config_from_account(ResellerId, Config).
 
--spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()} | {error, any()}.
+-spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()}.
 load_config_from_system(_Account, Config) ->
-    {'ok', kz_json:get_value(<<"default">>, maybe_new(kapps_config:get_category(Config)))}.
+    {'ok', kz_json:get_value(<<"default">>, kapps_config:get_category(Config), kz_json:new())}.
```

Comment:
"were the schemas generated with make apis ?" - no, schemas created and uploaded to system_schemas db manually. But this is not the reason I believe.

get_config calls 4 funs:
    Programm = [fun load_config_from_account/2
               ,fun load_config_from_reseller/2
               ,fun load_config_from_system/2
               ,fun load_default_config/2
               ],
https://github.com/2600hz/kazoo/blob/master/core/kazoo_config/src/kapps_config_util.erl#L17-L24

and load_config_from_system/2 gives 'undefined'
(kazoo_apps@sip.okui.info)6> kapps_config_util:load_config_from_system(<<>>, <<"paypal">>).
{ok,undefined}

it gives 'undefined' just because it looks directly into system_config db using kapps_config:get_category/1 (kz_datamgr:open_cache_doc/3) for a doc, which is absent and recieves {error,not_found} which becomes 'unefined' at the end of the day..